annotate.py

```diff
@@ -188,20 +207,36 @@ def annotate_img(img_path, initials, size1=1, size2=1) :
     file.close   
 
 ###############################Image Output##########################################
+    #Make mask same colour as drawing and output binarised image
+
+
     #Make mask same colour as drawing and output binarised image
     train_labels = img[:,:,:3]
-    # print("TRAIN LABELS = ")
-    #print(train_labels)
-    lower = np.array([0,0,254], dtype = "uint16")
-    upper = np.array([0,0,255], dtype = "uint16")
-    mask = cv2.inRange(train_labels, lower, upper)
-    mask[mask < 250] = 0
-    mask[mask != 0 ] = 255 ##set =1 for binary or =255 for visible white
+    lower_blue = np.array([254,0,0], dtype = "uint16")
```

Comment:
These numbers are hardcoded and repeated above in `draw_circle` (i.e. need two places to modify if changing, which can lead to mistakes). Find a way to use a single variable here and above (probably `global` is fine for this small script, though generally bad practice).

Reply:
Added global variables in the new version of PR